pkg/kubeclient/kubelet_eviction.go

```diff
@@ -0,0 +1,318 @@
+package kubeclient
```

Comment:
Please put a comment here too similar to previous work which can reference from where is this code copied over

Reply:
done